openreview/conference/templates/areachairWebfield.js

```diff
@@ -228,16 +228,21 @@ var formatData = function(blindedNotes, officialReviews, metaReviews, noteToRevi
   });
 };
 
+var displayError = function(message) {
```

Comment:
Is this function being used? I can't find a reference to it.

Reply:
This is being used in case of failure of getUserProfiles.